versions.tf

```diff
@@ -27,9 +27,9 @@ terraform {
     }
   }
   provider_meta "google" {
-    module_name = "blueprints/terraform/terraform-google-cloud-armor/0.1.0"
+    module_name = "blueprints/terraform/terraform-google-cloud-armor/0.3.0"
```

Comment:
Missing a "v" in the version no. i.e.:

`module_name = "blueprints/terraform/terraform-google-cloud-armor/v0.3.0"`

Same for the one below